.travis.yml

```diff
@@ -0,0 +1,16 @@
+language: node_js
+
+cache:
+    directories:
+        - node_modules
+
+node_js:
+  - '4'
+  - '5'
+  - '6'
+
+install:
+  - npm install
+
+script:
+  - npm test
```

Comment:
Does it affect anything if we do `npm test` (Like here) or `npm run test` (Like in the README)? They seem to work the same